Makefile

```diff
@@ -220,9 +221,12 @@ cni-plugins: get-cni-sources .out-stamp
 		"amazon/amazon-ecs-build-cniplugins:make"
 	@echo "Built amazon-ecs-cni-plugins successfully."
 
-run-integ-tests: test-registry gremlin container-health-check-image
+run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tests
```

Comment:
I'd prefer to keep the sudo business separated

Reply:
Since sudo tests are nothing but integ tests, I opted to keep it as part of `run-integ-tests`.